WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanViewController.swift

```diff
@@ -355,6 +372,13 @@ extension JetpackScanViewController: NoResultsViewControllerDelegate {
             static let imageName = "jetpack-scan-state-error"
         }
 
+        struct vaultPressError {
```

Comment:
[nit]

```suggestion
        struct VaultPressError {
```

Reply:
@hassaanelgarem I wrote it this way because the other `struct` also follows the same pattern. Given it's not a big deal I'll move forward with merging it because, otherwise, I'll have to change the others. Hope that's ok!